python/lsst/afw/cameraGeom/cameraGeomEnumDicts.py

```diff
@@ -23,9 +23,10 @@
 from . import cameraGeomLib
 
 DetectorTypeValNameDict = {
-    cameraGeomLib.SCIENCE:   "SCIENCE",
-    cameraGeomLib.FOCUS:     "FOCUS",
-    cameraGeomLib.GUIDER:    "GUIDER",
+    cameraGeomLib.SCIENCE: "SCIENCE",
+    cameraGeomLib.FOCUS: "FOCUS",
+    cameraGeomLib.GUIDER: "GUIDER",
     cameraGeomLib.WAVEFRONT: "WAVEFRONT",
 }
-DetectorTypeNameValDict = {val: key for key, val in DetectorTypeValNameDict.items()}
+DetectorTypeNameValDict = {val: key for key,
+                           val in DetectorTypeValNameDict.items()}
```

Comment:
I don't like this line break because `key, val` seem like they should be kept together for readability. Can it split after the `in` instead?